Resources/Textures/Clothing/Shoes/Boots/fisherboots.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprites made by RiceMar",
```

Comment:
https://github.com/RiceMar doesn't seem to have any repos?

Reply:
https://github.com/RiceMar1244

She personally typed out the copyright meta, and the changelog for their pull (the back-equippable guitars) had her name listed as RiceMar. I can ask her to change it if necessary.

Reply:
Should point to the github user so github.com/RiceMar1244 or some variant as we also have discord handles that get used.

Reply:
Forgot to write here - it uses her proper GitHub username now.